### ISSUE TITLE
An admin panel for configuration

### ISSUE THREAD

**Issue by Alwahsh** [glittergallery/GlitterGallery#216]
There are some cofiguration options that need to be added.
see #201 and the discussion on #214.
[0_glitterconfig.rb](https://github.com/glittergallery/GlitterGallery/blob/master/config/initializers/0_glitterconfig.rb)

I'm thinking of introducing an admin panel to allow for these configurations as it would be easier and more convenient for admins rather than a configuration file.
It can also be extended to allow for options like deleting any inappropriate content, banning users and other things an admin would like to be able to do.
This would need authorization and many edits to various places to be efficient.
Is that a good idea or an overkill?

**Comment by sarupbanskota** [comment]
yup, that's something definitely needed - although IMO we might not want to put all of our focus to it at this point. Personally, I would like to see this come after we're through with https://github.com/glittergallery/GlitterGallery/issues/161 and https://github.com/glittergallery/GlitterGallery/issues/162. 

Reason being we'll be administering the first real deployment and therefore will understand the configuration - agreed, it's tedious to have to meddle with config files versus clicking buttons; but priorities :) 

what do the others think? 

by all means, if you think this makes learning the app easier for you though, please take it up.

**Comment by mecyborg** [comment]
This should be done. It will help in managing.

**Comment by EmilyDirsh** [comment]
Agree with @sarupbanskota - this is something we'll need eventually, but IMO lower priority than a lot of other enhancements. Any site admins will have to deal with installing the app and a database, and configuration is part of that. A GUI can make it easier, but I'd rather see user-facing enhancements atm, rather than site admin enhancements.

**Comment by rohitpaulk** [comment]
^^ +1.
